src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Intervention.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import org.hibernate.annotations.CreationTimestamp
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.CascadeType
+import javax.persistence.Entity
+import javax.persistence.FetchType
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.OneToOne
+import javax.validation.constraints.NotNull
+
+@Entity
+data class Intervention(
+  @NotNull @Id @GeneratedValue val id: UUID? = null,
+  @NotNull @CreationTimestamp val createdAt: OffsetDateTime? = null,
+
+  @NotNull val title: String,
+  @NotNull val description: String,
+
+  @NotNull @OneToOne(fetch = FetchType.EAGER, cascade = [CascadeType.ALL]) @JoinColumn(name = "contract_id")
```

Comment:
what's the reason for eager fetching on this field?

Reply:
Wouldn't we always want to retrieve a contract when an intervention is retrieved?  There is so much information that the intervention relies upon.

Reply:
yes do not want the intervention to remove the contract - changed mapping